pos/src/main/java/edu/illinois/cs/cogcomp/pos/lbjava/POSTagger.java

```diff
@@ -75,6 +77,13 @@ public String discreteValue(Object __example) {
         return taggerUnknown.discreteValue(w);
     }
 
+    // TODO(Yewen): find a better way to extract the labels from the classifier.
+    // we tried to extract labels from LabelLexicon, but needed non-trivial amount of post-processing to get the raw
+    // label strings.
+    public Set<String> getTagValues() {
+        return taggerKnown.scores(new Token()).values();
```

Comment:
Yeah. Two ways to get actual tags:

Something like:
```
Lexicon labelLexicon = taggerKnown.getLabelLexicon();
Set<String> tagSet = new HashSet();
for (int i =0; i < labelLexicon.size(); ++i) {
    tagSet.add(labelLexicon.lookupKey(i).getStringValue());
}
```

or something like:
```
Set<Feature> labelFeatures = taggerKnown.getLabelLexicon().getMap().keySet();
```
and get the string value for each of the label features.